state-chain/runtime/src/migrations/arbitrum_integration.rs

```diff
@@ -87,36 +91,43 @@ impl OnRuntimeUpgrade for ArbitrumIntegration {
 				address_checker_address,
 				chain_id,
 				usdc_address,
-			): (Address, Address, Address, u64, Address) =
+				start_block_number,
+			): (Address, Address, Address, u64, Address, u64) =
 		match cf_runtime_upgrade_utilities::genesis_hashes::genesis_hash::<Runtime>() {
 			cf_runtime_upgrade_utilities::genesis_hashes::BERGHAIN =>
 			{
+				panic!("Need to set up arbitrum integration for Berghain");
```

Comment:
these are so we don't forget to set these to the real values when we're ready to set them